### ISSUE TITLE
Profile settings not imported at headless mode

### ISSUE THREAD

**Issue by prometheus20416** [gologinapp/gologin#19]
Hi,
When headless enabled go login profile settings not imported.

    (async () =>{
        const GL = new GoLogin({
            token: '',
            profile_id: '',
            extra_params: ['--headless'],
        });
        const {status, wsUrl} = await GL.start(); 
        const browser = await puppeteer.connect({
            browserWSEndpoint: wsUrl.toString(), 
            ignoreHTTPSErrors: true,
        });
    const page = await browser.newPage();
    const viewPort = GL.getViewPort();
    await page.setViewport({ width: Math.round(viewPort.width * 0.994), height: Math.round(viewPort.height * 0.92) });
    const session = await page.target().createCDPSession();
    const { windowId } = await session.send('Browser.getWindowForTarget');
    await session.send('Browser.setWindowBounds', { windowId, bounds: viewPort });
    await session.detach();
    
    await page.goto('https://iphey.com'); 
    await new Promise((resolve) => setTimeout(resolve, 20000));
  
    await page.screenshot({path: 'iphey.jpg', fullPage: true});

    await browser.close();
    await GL.stop();
    })();

This is the ss result : 
![ss](https://user-images.githubusercontent.com/88255323/136867736-b5adcd9b-824e-45a5-8ae3-3f52d50f36b1.png)

**Comment by pango-lier** [comment]
Do you have a solution to this problem? 
I had the same problem ?
i use linux ubuntu OS and orbita-browser

**Comment by Mixolap** [comment]
try add delay after browser.newPage()

**Comment by OleksandrKorzhov** [comment]
Any solution to this? Have the same issue. Or is there a way to run the GoLogin browser in a docker container without headless mode?
